storybook/src/pages/amopis/DetailsPage/DetailsPage.docs.mdx

```diff
@@ -0,0 +1,12 @@
+{/* @license CC0-1.0 */}
+
+import { Meta, Primary } from "@storybook/blocks";
+import * as DetailsPageStories from "./DetailsPage.stories.tsx";
+
+<Meta of={DetailsPageStories} />
+
+# Details Page
+
+Displays properties of an entity.
+
+<Primary />
```

Comment:
This will display a page that is too narrow, maybe we can write these page examples without Canvas and just have some code examples. Storybook has a `Source` component to include some source files.